owtf/db/transaction_manager.py

```diff
@@ -217,7 +218,9 @@ def get_transactions(self, transactions):
         """
         owtf_tlist = []
```

Comment:
Wouldn't it be possible to shorten those lines with:
```python
return [self.get_transaction(transaction) for transaction in transactions if self.get_transaction(transaction) is not None]
```

Reply:
+1